src/api/app/views/webui2/webui/projects/rebuild_times/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,7 @@
+= render partial: 'webui/main/breadcrumb_items'
```

Comment:
I believe it's better if we build on top of the project's breadcrumb items. This way, we could remove the lines 2 to 5 (which are duplicated).

So this would be: `= render partial: 'webui/project/breadcrumb_items'`

Reply:
> I believe it's better if we build on top of the project's breadcrumb items. This way, we could remove the lines 2 to 5 (which are duplicated).
> 
> So this line would be: `= render partial: 'webui/project/breadcrumb_items'`

Done.